### PR TITLE
Adding Basic Calculator

### PR DESCRIPTION
**Pull request by abhishekbansal276**
I have added basic calculator class. Please accept my PR and give me feedback.

### CODE REVIEW

File: src/main/scala/abhishek_bansal/basic_calculator/BasicCalculator.scala [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+sealed trait ErrorInCalculations
+
+case class DivisionByZeroError(message: String) extends ErrorInCalculations
+
+case class InvalidOperationError(message: String) extends ErrorInCalculations
+
+class BasicCalculator {
+  def calculate(value_1: Double, value_2: Double, operation: String): Either[ErrorInCalculations, Double] = {
+    operation match {
+      case "Addition" => Right(value_1 + value_2)
```
Comment (tg-trainer) [inline src/main/scala/abhishek_bansal/basic_calculator/BasicCalculator.scala:12]: variable name should be in lowerCamelCase case only

**Review by tg-trainer — CHANGES_REQUESTED** [review]
Please update the name at all the places

File: src/main/scala/abhishek_bansal/basic_calculator/BasicCalculator.scala [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      case "Subtraction" => Right(valueOne - valueTwo)
+      case "Multiplication" => Right(valueOne * valueTwo)
+      case "Division" => division(valueOne, valueTwo)
+      case "Modulus" => modulus(valueOne, valueTwo)
+      case _ => Left(InvalidOperationError("This operation is not added till now (Please choose between Addition, Subtraction, Multiplication, Division, Modulus"))
+    }
+  }
+
+  private def division(value1: Double, value2: Double): Either[ErrorInCalculations, Double] = {
+    if (value2 != 0) Right(value1 / value2)
```
Comment (tg-trainer) [inline src/main/scala/abhishek_bansal/basic_calculator/BasicCalculator.scala:22]: don't use the number

**Review by tg-trainer — CHANGES_REQUESTED** [review]
Let me know when you are done

File: src/main/scala/abhishek_bansal/basic_calculator/BasicCalculator.scala (hunk 2 above)
Comment (abhishekbansal276) [reply src/main/scala/abhishek_bansal/basic_calculator/BasicCalculator.scala]: done sir

**Pull request merged**